### ISSUE TITLE
Working with Weekly or Montly time series

### ISSUE THREAD

**Issue by Akaori** [awslabs/gluon-ts#78]
Hello,

I am having difficulties to work with Weekly and Montly frequency.

When I run this code using Daily frequenct, it works properly, however, if I resample to weekly or montly I get the following error:

`TypeError: unsupported operand type(s) for *: 'int' and 'NoneType'`

Here is my code:

```
from gluonts.dataset.util import to_pandas
from gluonts.trainer import Trainer
import numpy as np
import matplotlib.pyplot as plt
plt.style.use('seaborn')
from gluonts.dataset import common
from gluonts.model import deepar
import pandas as pd

!pip install quandl

import quandl

dataset = pd.DataFrame(quandl.get("FRED/DEXBZUS"))

dataset = dataset.resample('M').apply(lambda ser: ser.iloc[-1,])

training_data = common.ListDataset([{"start": dataset.index[0],
                            "target": dataset.Value[:"2017"]}],
                          freq="M")

estimator = deepar.DeepAREstimator(freq="M", prediction_length=40, trainer=Trainer( epochs=5))
predictor = estimator.train(training_data=training_data)

for test_entry, forecast in zip(training_data , predictor.predict(training_data )):
    to_pandas(test_entry)[-60:].plot(linewidth=2)
    forecast.plot(color='g', prediction_intervals=[50.0, 90.0])
plt.grid(which='both')
```

**Comment by vafl** [comment]
Thanks for reporting this. It should be working now. Can you try again?

**Comment by Akaori** [comment]
Hello @vafl,

It continus to appears this error:

```
TypeError                                 Traceback (most recent call last)
<ipython-input-3-1a2106d0838b> in <module>()
     20 
     21 estimator = deepar.DeepAREstimator(freq="M", prediction_length=40, trainer=Trainer( epochs=5))
---> 22 predictor = estimator.train(training_data=training_data)
     23 
     24 for test_entry, forecast in zip(training_data , predictor.predict(training_data )):

15 frames
/usr/local/lib/python3.6/dist-packages/gluonts/transform.py in _compute_date_helper(ts, freq, offset)
    100         # bounds values over year 9999 raise a ValueError
    101         # values over 2262-04-11 raise a pandas OutOfBoundsDatetime
--> 102         return ts + offset * ts.freq
    103     except (ValueError, pd._libs.OutOfBoundsDatetime) as ex:
    104         raise GluonTSDateBoundsError(ex)

TypeError: unsupported operand type(s) for *: 'int' and 'NoneType'
```

**Comment by lostella** [comment]
Hi @Akaori, the fix is still in the `master` branch only. To use that you must first uninstall gluonts, then clone this git repo, cd into the directory and
```
pip install .
```
Your `--upgrade` option will work only once a new release containing the fix will be done.
Hope the fix works!

**Comment by Akaori** [comment]
Hi @lostella, thanks for reply.

I tryied run on google colab:

```
!pip uninstall gluonts
!git clone https://github.com/awslabs/gluon-ts.git
!pip install gluonts
```
But doesn't work.

Then I tried:
```
!pip uninstall gluonts
!pip install git+https://github.com/awslabs/gluon-ts.git
```
The error still persists.

Probably I am not installing the master branch repo correcly.

**Comment by gjmulder** [comment]
I'm getting the same issue with a pull from master:

```
commit ac782de64e6ef0b3b6bb43eb7d5a9da5689f8886 (HEAD -> master, origin/master, origin/HEAD)
Author: Lorenzo Stella <lorenzostella@gmail.com>
Date:   Thu Oct 17 11:27:09 2019 +0200
```

**Comment by lostella** [comment]
@gjmulder the original MWE of this issue works fine for me, can you provide a new one that raises the error you're getting?

**Comment by jaheba** [comment]
Could this be an issue related to the pandas version used?

@gjmulder can you run `pip show pandas`? It should be at least v0.25

https://github.com/awslabs/gluon-ts/blob/ac782de64e6ef0b3b6bb43eb7d5a9da5689f8886/requirements/requirements.txt#L6

**Comment by gjmulder** [comment]
After a completely clean install and re-download of the M4 data I can't reproduce the issue.

I suspect it was an issue with my M4 metadata. I had incorrectly manually added the freq metadata parameter but set it to int instead of the Pandas frequency string, due to another issue I encountered.
